docs/src/content/docs/fr/resources/plugins.mdx

```diff
@@ -93,6 +93,16 @@ Les [modules d'extension](/fr/reference/plugins/) peuvent personnaliser la confi
 		title="starlight-site-graph"
 		description="Ajoutez un graphe interactif de votre site dans la barre latérale de votre page."
 	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-sidebar-topics"
+		title="starlight-sidebar-topics"
+		description="Divisez votre documentation en différentes sections, chacune ayant sa propre barre latérale."
+	/>
+	<LinkCard
+		href="https://github.com/trueberryless-org/starlight-sidebar-topics-dropdown"
+		title="starlight-sidebar-topics-dropdown"
+		description="Divisez votre page de documents en plusieurs sous-pages et passez de l'une à l'autre à l'aide d'un menu déroulant dans la barre latérale."
```

Comment:
```suggestion
		description="Divisez vos pages de documentation en plusieurs sous-pages et passez de l'une à l'autre à l'aide d'un menu déroulant dans la barre latérale."
```

I think "votre page de documents" may be a bit weird? I think the intent is to reference multiple pages of a documentation.

Reply:
Oh yeah it's better! Thanks!